src/sentry/static/sentry/app/components/group/suggestedOwnerHovercard.jsx

```diff
@@ -158,7 +185,16 @@ const CommitDate = styled(({date, ...props}) => (
   color: ${p => p.theme.gray2};
 `;
 
-const ReasonItem = styled('div')`
+const CommitReasonItem = styled('div')`
+  display: flex;
+  align-items: flex-start;
```

Comment:
Is this default?

Reply:
Default is stretch, without this tags are stretched, when children of this are taller than 1 lh:
![Screen Shot 2020-05-22 at 3 42 11 PM](https://user-images.githubusercontent.com/15015880/82714335-f4f9af00-9c42-11ea-92ab-08f0dbc42133.png)